src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -310,6 +315,14 @@ public void setFirestoreQueryBatchSize(int firestoreQueryBatchSize) {
         this.firestoreQueryBatchSize = firestoreQueryBatchSize;
     }
 
+    public int getMaxDrsLookups() {
+        return maxDrsLookups;
+    }
+
+    public void setDrsLookupMax(int maxDrsLookups) {
```

Comment:
Need to update this too so the getter/setters match
```suggestion
    public void setMaxDrsLookups(int maxDrsLookups) {
```